product_brand/__manifest__.py

```diff
@@ -1,25 +1,27 @@
-# -*- coding: utf-8 -*-
 # © 2009 NetAndCo (<http://www.netandco.net>).
 # © 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com>
 # © 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com>
 # © 2016 Serpent Consulting Services Pvt. Ltd.
 # © 2018 Daniel Campos <danielcampos@avanzosc.es>
+# Copyright 2018 Tecnativa - David Vidal
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
 {
     'name': 'Product Brand Manager',
-    'version': '11.0.1.0.2',
+    'version': '11.0.1.1.0',
     'category': 'Product',
     'summary': "Product Brand Manager",
-    'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
-              ', MONK Software, Odoo Community Association (OCA)'
-              ', SerpentCS Pvt. Ltd.',
+    'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA, '
+              'MONK Software, SerpentCS Pvt. Ltd., Tecnativa, '
```

Comment:
Make it Serpent Consulting Services Pvt. Ltd. (so its added to right author on search)